index.js

```diff
@@ -18,8 +18,15 @@ var Promise     = require( 'es6-promise' ).Promise;
  *
  * @tested
  */
-function readDirectory( projectPath, credits ) {
+function readDirectory( projectPath, credits, seen ) {
   credits = credits || [];
+  seen = seen || [];
```

Comment:
Can we align the `=`?  
I'm missing the eslint settings for this obviously.

After that we're ready to go. :)